lib/world_cup.rb

```diff
@@ -0,0 +1,59 @@
+class WorldCup
+  attr_reader :year, :teams
+  def initialize(year, teams)
+    @year = year
+    @teams = teams
+  end
+
+  def active_players_by_position(position)
+    @teams.map do |team|
+      if team.eliminated? == false
+        team.players.find_all do |player|
+          player.position == position
+        end
+      else
+        []
+      end
+    end.flatten
+  end
+
+  # def all_players(boolean)
```

Comment:
I REALLY like your start on this - love that you broke this logic up into multiple helper methods!!